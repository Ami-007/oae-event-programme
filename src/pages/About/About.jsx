import React from "react";
import Navbar from "../../components/Navbar/Navbar";
import Photo from "../../components/Photo/Photo";

const About = () => {
  return (
    <div>
      <h1>About page</h1>
      <Photo
        imageSrc="https://place-puppy.com/300x300" alt="puppy"
        description="A picture of a man's best friend"/>
      <Navbar/>
    </div>
  );
};

export default About;
