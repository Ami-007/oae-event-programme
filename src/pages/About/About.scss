@use "../../assets/sass/variables" as *;

.about {
  &__content {
    padding: 0 2.5rem;
  }
  &__callout {
    color: var(--theme-color-text);
    font-family: $modern-header;
    display: flex;
    justify-content: flex-end;
    padding: 0 2.5rem 0 2.5rem;
  }
}
