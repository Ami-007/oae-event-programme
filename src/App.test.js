import { render } from "@testing-library/react";
//import App from "./App";
import About from "./pages/About/About"

// test("Renders App component", () => {
//   const { container } = render(<App />);
//   expect(container).toMatchSnapshot();
// });

test("Renders About component, photo and description should appear", () => {
  const { container } = render(<About />);
  expect(container).toMatchSnapshot();
});
