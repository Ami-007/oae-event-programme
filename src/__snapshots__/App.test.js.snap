// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders App component with default page 1`] = `
<div>
  <div
    class="home"
  >
    <h1
      class="home__header"
    >
      Home page
    </h1>
    <div
      class="header-home"
    >
      <h1
        class="header-home__title"
      >
        BACH, THE UNIVERSE & EVERYTHING
      </h1>
      <span
        class="header-home__subtitle"
      >
        Can You Hear The Shape of The Drum?
      </span>
    </div>
    <div
      class="event-header-details"
    >
      <p
        class="event-header-details__time-date"
      >
        11.30am
        <span
          class="event-header-details__divider"
        >
          |
        </span>
        Sun 21 Nov 2021
      </p>
      <p
        class="event-header-details__venue-location"
      >
        <span
          class="event-header-details__venue"
        >
          South Bank
        </span>
        <span
          class="event-header-details__comma"
        >
          ,
        </span>
         
        London
      </p>
    </div>
    <iframe
      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
      allowfullscreen=""
      class="youtube-embed"
      frameborder="0"
      src="https://www.youtube.com/embed/q6EoRBvdVPQ"
      title="Embedded youtube"
    />
    <p
      class="paragraph"
      data-testid="paragraph"
    >
      this is some paragraph
    </p>
    <a
      class="view-schedule-button"
      href="/event-schedule"
    >
      View Schedule
    </a>
  </div>
</div>
`;
